api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -78,7 +75,7 @@ public static OpenTelemetry get() {
    * <p>If you are using the OpenTelemetry SDK, you should generally use {@code
    * OpenTelemetrySdk.builder().buildAndRegisterGlobal()} instead of calling this method directly.
    */
-  public static void set(OpenTelemetry openTelemetry) {
+  public static OpenTelemetry set(OpenTelemetry openTelemetry) {
```

Comment:
should we add javadoc about the `@return` on here now? This API is slightly strange to simply return exactly what is passed in.

Reply:
Doh, sorry I thought this was a private method so tried to save a line of code. Let me revert this